packages/react-google-maps-api/src/components/maps/BicyclingLayer.tsx

```diff
@@ -3,26 +3,15 @@ import { PureComponent } from 'react'
 
 import MapContext from '../../map-context'
 
-import { BicyclingLayerPropTypes } from '../../proptypes'
-
 export class BicyclingLayer extends PureComponent {
-  static propTypes = BicyclingLayerPropTypes
-
   static contextType = MapContext
 
   state = {
     bicyclingLayer: null
   }
 
   componentDidMount = () => {
-    const bicyclingLayer = new google.maps.BicyclingLayer(
-      Object.assign(
-        {
-          map: this.context
-        },
-        this.props.options
-      )
-    )
```

Comment:
And here ))))

Reply:
Bicycling layer constructor doesn't accept options property